DataConnectors/Qualys VM/README.md

```diff
@@ -0,0 +1,16 @@
+# Qualys Host Detections
```

Comment:
Thanks, for putting this together. 
We are working on landing this data connector in Azure Sentinel along with documentation too.  Once that happens you might want to point this README directly to detailed instructions in doc / directly get this data connector enabled within Azure Sentinel.